tests/k6/scenarios/core2_qa_control_plane_ops.js

```diff
@@ -0,0 +1,206 @@
+/**
+ * This test aims to verify that Core 2 remains in an error-free state after
```

Comment:
Nit: this scenario focuses only on models so far, should this reflected in its name or should this will be extended for pipelines and experiments as well?

Reply:
I was thinking I will extend this for pipelines/experiments. But happy to have them as separate scenarios if we want to split things up. At the moment the pipelines tests are quite weak (just single model pipelines).